flake/hermes.nix

```diff
@@ -0,0 +1,39 @@
+{ self, ... }: {
+  perSystem = { self', pkgs, systemCommonRust, subnix, lib, system, ... }:
+    let
+      devnet-root-directory = "/tmp/composable-devnet";
+      validator-key = "osmo12smx2wdlyttvyzvzg54y2vnqwq2qjateuf7thj";
+    in {
+      packages = rec {
+        hermes = self.inputs.cosmos.packages.${system}.hermes_1_5_1;
+        osmosis-centauri-hermes-init = pkgs.writeShellApplication {
+          runtimeInputs = [ hermes ];
+          name = "osmosis-centauri-hermes-init";
+          text = ''
+            HOME=$(realpath .)
+            MNEMONIC_FILE="$HOME/.hermes/mnemonics/relayer.txt"
+            export HOME
+            mkdir --parents "$HOME/.hermes/mnemonics/"
+
+            echo "black frequent sponsor nice claim rally hunt suit parent size stumble expire forest avocado mistake agree trend witness lounge shiver image smoke stool chicken" > "$MNEMONIC_FILE"
+            hermes keys add --chain centauri-dev --mnemonic-file "$MNEMONIC_FILE" --key-name centauri-dev --overwrite
```

Comment:
Why not use hyperspace?

Reply:
main goal was to make 100% sure it runs for now and hopefully inits as fast as possible.